workflows-cli/template-boilerplate/conventional-templates/hello-world.yaml

```diff
@@ -0,0 +1,69 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/argoproj/argo-workflows/main/api/jsonschema/schema.json
+
```

Comment:
I would add somewhere (maybe a README.md in the root directory?) a link directing the user to the workflows docs, https://diamondlightsource.github.io/workflows/docs/, telling them that they will find instructions on how to write templates there.